channels/us.m3u

```diff
@@ -784,15 +784,14 @@ https://rpn1.bozztv.com/36bay2/gusa-chefrock/mono.m3u8
 https://a.jsrdn.com/broadcast/4df1bf71c1/+0000/c.m3u8
 #EXTINF:-1 tvg-id="ChiveTV.us" tvg-name="Chive TV" tvg-country="US" tvg-language="" tvg-logo="https://od.lk/s/MF8yMzE5OTIyMjBf/Redbox_512x512.png" group-title="",Chive TV
 https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxchive/CDN/playlist.m3u8
-#EXTINF:-1 tvg-id="ChopperTown.us" tvg-name="Chopper Town" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Chopper Town
-https://6c4d10b7b847421db903914a39b00eb7.mediatailor.us-west-2.amazonaws.com/v1/manifest/ba62fe743df0fe93366eba3a257d792884136c7f/LINEAR-11-CHOPPERTOWN-PLEX/67d41d7e-f2c4-4005-abab-22174f1f8b43/0.m3u8
```

Comment:
Something went wrong here

Reply:
Is this channel link removed intentionally?

Reply:
On which aspect : the stream or the fact that it's twice on the playlist ?

Because I've reviewed that part when i've made the "big cleaning", and I don't reckon having this twice.

Reply:
No, I think I caused that on my end. There were two rows above the URL link, so I removed one of them. Not the entire channel, only the double listing of the #EXTINF: string and everything after it.

Reply:
Well it seems it either lagged or something wrong happened. I'm gonna try to reinstate the link.